UPGRADING.md

```diff
@@ -0,0 +1,56 @@
+UPGRADING
+--------
+
+This file documents how to upgrade an existing BHIMA installation to a new release.  To learn about the releases process, see [RELEASES](./RELEASES.md).
+
+Upgrading an existing installation takes three steps:
+
+1. Preparing the BHIMA software
+2. Preparing the database
+3. Running the upgrade
+
+Steps 1 and 2 can happen in parallel and can be done at a leasurely pace.  The third step is destructive and will commit the changes, deploying the new version at the location.
+
+These steps describe the typical deployment setup.  We assume that the BHIMA software is deployed in `$HOME/apps/`, `$VERSION` will be the next version name, and will refer to `$DATABASE` as the database name.
+
+## Step 1: Peparing the BHIMA software
+
+1. Go to https://github.com/ima-worldhealth/bhima/releases/latest in your web browser.
+2. Copy the link to the `$VERSION.tar.gz` file.
+2. Connect to the server with SSH
+3. Start a `screen` session in case the connection drops during the upgrade.
+4. Change directory into `$HOME/apps/`.
+5. Download the latest release with `wget -c "$URL"`.  This will create `$VERSION.tar.gz` in the `$HOME/apps/` directory.
+6. Unzip the release file with `tar xf $VERSION.tar.gz`.  This will create the folder `bhima-$RELEASE/` in the `$HOME/apps/` directory.
+7. Copy the `.env` file from the current application into the `bhima-$RELEASE/bin/` folder and make any changes necessary to `.env`.
+8. Copy the contents (but not sub-directories) from the `bhima-$RELEASE/bin/` folder into the `bhima-$RELEASE/` folder.  This can be done with `cp $HOME/apps/bhima-$RELEASE/bin/* $HOME/apps/bhima-$RELEASE/bin/`
+9. Change directory into the release directory with `cd $HOME/apps/bhima-$RELEASE/`
+10. Install the `node_modules/` in the `bhima-$RELEASE/` folder with the command `NODE_ENV=production yarn`.
```

Comment:
I suggest when you end line with a command, just omit the trailing period.  It is to easy to assume that you should type the period as part the command.   Instead, you can end the sentence part with a colon, eg:

      ... folder with the command: 'NODE_ENV=production yarn'

Reply:
Alright, will do